### ISSUE TITLE
Is this hook still working?

### ISSUE THREAD

**Issue by dohomi** [will-t-harris/use-instagram-feed#2]
Hello

there seem some public changes on the Instagram API. Does this hook still work for you? I receive a CORS error when I try to use it.

Cheers

**Comment by will-t-harris** [comment]
Hi @dohomi 

I haven't tried to use the hook in quite some time, so I wouldn't be surprised if API changes have killed the approach. It felt pretty hacky to begin with.

I'd be open to PRs, though my understanding is that Instagram has made it more difficult to programmatically scrape posts without some sort of auth token.

**Comment by dohomi** [comment]
@will-t-harris yeah the issue is pretty new, probably for very few weeks like 1/2 weeks. I am looking for a solution right now will keep you posted -I actually don't use this plugin but I just found out that you have the same approach

**Comment by will-t-harris** [comment]
Any update on this @dohomi ?

**Comment by dohomi** [comment]
@will-t-harris no.. I am quite frustrated and will need to write a different scraper I am afraid

**Comment by eduanmoldeep** [comment]
it seems the approach is long blocked, rather find ways to use the approach with auth token.
Just an advice for whoever visits next.
